components/range_cache_memory_engine/src/range_stats.rs

```diff
@@ -45,6 +43,9 @@ pub(crate) struct RangeStatsManager {
     mvcc_amplification_threshold: usize,
 
     mvcc_amplification_record: Arc<Mutex<HashMap<u64, f64>>>,
+
+    load_evict_interval: Duration,
```

Comment:
Will `load_evict_interval` and `mvcc_amplification_threshold` support dynamic configuration?

Reply:
Good catch，I have ignored this.

Reply:
I think the code before this PR does not support dynamic configuration for load_evict_interval but does not return errors when user trys to change it. I will propose another PR to fix all cases like this.